packages/sdk/src/evm/constants/urls.ts

```diff
@@ -296,7 +296,13 @@ export function getProviderFromRpcUrl(
                   headers,
                   skipFetchSetup: _skipFetchSetup,
                 },
-                chainId,
+                chainId === Base.chainId
+                  ? {
+                      name: Base.name,
+                      chainId: Base.chainId,
+                      ensAddress: Base.ens.registry,
```

Comment:
basically here since you're already doing a special case for Base - might as well pass the ens resolver address as a const directly here

Reply:
Thanks @joaquim-verges for reviewing my pr. That is good to know that network info will get overriden with remove sync. I will hardcode it for now and update the pr shortly.